R/get.R

```diff
@@ -86,7 +89,12 @@ get_users <- function(src, page_size = 500, prefix = NULL, limit = Inf) {
 get_groups <- function(src, page_size = 500, prefix = NULL, limit = Inf) {
   validate_R6_class(src, "Connect")
 
-  res <- page_offset(src, src$groups(page_size = page_size, prefix = prefix), limit = limit)
+  if (!is.null(prefix)) {
+    response <- src$groups(page_size = page_size, prefix = prefix)
+    res <- response$results
+  } else {
+    res <- page_offset(src, src$groups(page_size = page_size, prefix = prefix), limit = limit)
```

Comment:
`prefix` here will always be `NULL` correct? Would it make sense to do `res <- page_offset(src, src$groups(page_size = page_size), limit = limit)` instead or possibly `res <- page_offset(src, src$groups(page_size = page_size, prefix = NULL), limit = limit)` if you're worried that the default of the `groups` method might change under you? (Though that's also exactly where the tests that Taylor asked about would help of course!)

Reply:
Yeah, either of those (and ensuring that this is covered by tests) would make sense.